tests/ci/ci_config.py

```diff
@@ -63,7 +63,7 @@
             "with_coverage": False,
         },
         "package_tsan": {
-            "compiler": "clang-14",
+            "compiler": "clang-13",
```

Comment:
As far as I understand we agreed to add such an option, not to replace the usage of clang-14

Reply:
It's a config for tsan builder only. Not quite get what you mean particularly.

All other builds are still using clang-14

Reply:
Here, [TSAN](https://github.com/ClickHouse/ClickHouse/runs/7457763103?check_suite_focus=true#step:7:505) is built with clang13, and anything else, for example [release](https://github.com/ClickHouse/ClickHouse/runs/7457762894?check_suite_focus=true#step:7:509) and [ASAN](https://github.com/ClickHouse/ClickHouse/runs/7457762487?check_suite_focus=true#step:7:505) are built with clang-14